build/helper/metadata_add_all.py

```diff
@@ -33,21 +33,12 @@ def _add_python_parameter_name(parameter):
     return parameter
 
 
-def _add_python_type(parameter):
-    '''Adds a python_type key/value pair to the parameter metadata'''
-    if parameter['enum'] is None:
-        parameter['python_type'] = parameter['type']
-    else:
-        parameter['python_type'] = 'enums.' + parameter['enum']
-    return parameter
-
-
 def _add_intrinsic_type(parameter):
     '''Adds a intrinsic (basic python type) key/value pair to the parameter metadata'''
     if parameter['enum'] is None:
         parameter['intrinsic_type'] = get_intrinsic_type_from_visa_type(parameter['type'])
     else:
-        parameter['intrinsic_type'] = parameter['python_type']
+        parameter['intrinsic_type'] = 'enums.' + parameter['enum']
```

Comment:
Why not call this 'python_type'?
That way every parameter has 2 types associated to it: the Visa one and the Python API one. Intrinsic is less clear in my opinion.

Or is this a temporary rename pending more changes?

Reply:
Next PR